sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeCustomizationTest.java

```diff
@@ -6,20 +6,25 @@
 package com.microsoft.appcenter.distribute;
 
 import android.app.Activity;
+import android.app.DownloadManager;
 import android.content.DialogInterface;
 
 import com.microsoft.appcenter.channel.Channel;
 import com.microsoft.appcenter.http.HttpClient;
 import com.microsoft.appcenter.http.ServiceCall;
 import com.microsoft.appcenter.http.ServiceCallback;
 import com.microsoft.appcenter.utils.AppCenterLog;
+import com.microsoft.appcenter.utils.AsyncTaskUtils;
 import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
 
 import org.junit.Test;
+import org.mockito.Mockito;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 
+import static android.os.AsyncTask.Status.FINISHED;
```

Comment:
sort imports - these should be up with the android imports

Reply:
Actually unused, I cannot stress enough that if commit dialog is not used, the cleanup actions need to be performed manually.